common/governments/00_government_types.txt

```diff
@@ -61,15 +61,9 @@ theocracy_government = {
 
```

Comment:
`00_faction_effects.txt` needs adaptation.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/104191568-19570580-5437-11eb-9f37-080e8829f6e1.png)

</details>

`00_decisions_effects.txt` needs adaptation.

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/104191752-59b68380-5437-11eb-8197-b600a0472e1a.png)

</details>

Recheck using of `clan_government` in game files.